hadoop-ozone/cli-admin/src/main/java/org/apache/hadoop/ozone/admin/om/UpdateRangerSubcommand.java

```diff
@@ -65,12 +64,6 @@ public class UpdateRangerSubcommand implements Callable<Void> {
 
   @Override
   public Void call() throws Exception {
-
-    if (StringUtils.isEmpty(omServiceId) && StringUtils.isEmpty(omHost)) {
-      System.err.println("Error: Please specify -id or -host");
-      return null;
-    }
```

Comment:
Now that the test passes the parameter, this change is no longer needed.  Please restore `UpdateRangerSubcommand`.